packages/experimental/Avatar/src/__tests__/JSAvatar.test.jsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { JSAvatar } from '..';
+import { getInitials, validateAlphabeticalCharacters, removeSuffixesFromName } from '../useAvatar';
+import * as renderer from 'react-test-renderer';
+
+it('renders Avatar', () => {
+  const tree = renderer
+    .create(
+      <JSAvatar active="active" activeAppearance="ring" size="size120" shape="circular" accessibilityLabel="Test" avatarColor="beige" />,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+describe('getInitials method', () => {
+  const alphabeticalTestNames = [
+    'Marie',
+    'Aadi KaPoor',
+    'Marie Beaudouin',
+    'Aadi Meni KaPoor',
+    'Aadi Meni Ka Jr',
+    'Will Little (Teams)',
+    'Will Little-Tanaka',
+    '-Aadi Kapoor',
+    'Richard 23 Feynman',
+    '耿 James', // maybe should be in another column
```

Comment:
I thought the spec said that this should result in an icon being displayed, no initials?

Reply:
Yes, I've added this test before out meeting :) I'll fix it

Reply:
In this case, this one should also fall back to icon? * Richard *
We have a phrase "If first character is not alphabetic, fall back to icon" and "Ignore special characters i.e. parantheses, dashes". What if the first character is a special character? In this case it's not alphabetic. Should it be also icon? I'll also ask EJ